packages/graphback-runtime/src/service/CRUDService.ts

```diff
@@ -74,10 +75,10 @@ export class CRUDService<T = any> implements GraphbackCRUDService<T>  {
         return result;
     }
 
-    public findAll(context?: any): Promise<T[]> {
-        this.logger.log(`querying object ${this.modelName}`)
+    public findAll(page?: GraphbackPage, context?: any): Promise<T[]> {
```

Comment:
This will not work without change in resolvers plugin and general spec. It worked as context was any